src/main/java/uk/ac/cam/cl/dtg/isaac/api/managers/GameManager.java

```diff
@@ -1235,14 +1243,9 @@ private IsaacWildcard getWildCardById(final String id) throws NoWildcardExceptio
             fieldsToMatchOutput.put(newEntry, gameFilter.getConcepts());
         }
 
+        // handle exclusions
         List<String> tagsToExclude = Lists.newArrayList();
-        // add no filter constraint
-        tagsToExclude.add(HIDE_FROM_FILTER_TAG);
-
-        // Temporarily ignore book and quick_quiz question types.
-        // Strings hardcoded until question types are passed in as part of the query.
-        tagsToExclude.addAll(ImmutableList.of("quick_quiz", "book"));
```

Comment:
Does this rely on content having tagged all of the book/ quick_quiz questions not as problem_solving? Or is it a 'whitelist' and it's fine as it defaults to just be the problem_solving tagged ones now. I couldn't get it to cause an issue so I think it's fine

Reply:
Before this pull request, we did not return anything from the filter if it was tagged `quick_quiz` or `book` to allow the content team to add subject and topic tags to the books/quick_quizes.
Now we're flipping it. The accompanying frontend PR will add a `questionCategory=problem_solving` to the gameboard query. If that is present we will only return questions tagged `problem_solving`. Eventually they will be able to set `questionCategory=book,quick_quizes` and return those types of questions but we're not surfacing that to the users yet.

This relies on the content team having tagged all problem solving questions with the tag `problem_solving` from my understanding, and from inspection, this is done. They can now go ahead and tag the maths questions as both `problem_solving` and `book` (they are currently only `problem_solving` because having `book` excluded them from the results before this PR).

Reply:
Sounds good - thanks for clarifying